### PR TITLE
Import cleanup

### PR DESCRIPTION
**Pull request by MarkDaoust**
FIx TypedDict imports, and add a test.

Fixes: https://github.com/google/generative-ai-python/issues/209

### CODE REVIEW

File: tests/test_typing_extensions.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+from absl.testing import absltest
+
+TYPING_RE = re.compile(r'\btyping\b.*TypedDict')
+
+class TypingExtensionsTests(absltest.TestCase):
+    def test_no_typing_typed_dict(self):
+        root = pathlib.Path(__file__).parent.parent
+        for fpath in root.rglob("*.py"):
+            print(fpath)
```
Comment (markmcd) [inline tests/test_typing_extensions.py:27]: intentional? `print` is noisy for tests but if it's helpful then I have no objections.

File: tests/test_typing_extensions.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import pathlib
+import re
+
+from absl.testing import absltest
+
+TYPING_RE = re.compile(r'\btyping\b.*TypedDict')
+
+class TypingExtensionsTests(absltest.TestCase):
+    def test_no_typing_typed_dict(self):
```
Comment (markmcd) [inline tests/test_typing_extensions.py:24]: Is there an exit-condition for this lint rule? If so we should document it here somewhere.

**Review by markmcd — APPROVED** [review]

File: tests/test_typing_extensions.py (hunk 2 above)
Comment (MarkDaoust) [reply tests/test_typing_extensions.py:34]: Done.

File: tests/test_typing_extensions.py (hunk 1 above)
Comment (MarkDaoust) [reply tests/test_typing_extensions.py]: Done.

**Review by shilpakancharla — APPROVED** [review]

**Pull request merged**